service/lib/agama/dbus/clients/manager.rb

```diff
@@ -54,7 +54,7 @@ def commit
         end
 
         def current_installation_phase
-          dbus_phase = dbus_object["org.opensuse.DInstaller.Manager1"]["CurrentInstallationPhase"]
+          dbus_phase = dbus_object["org.opensuse.Agama1.Manager"]["CurrentInstallationPhase"]
```

Comment:
watch out, the autorename did not work right here

it should be `Agama.Manager1` for consistency with the other interfaces